superset-frontend/src/explore/components/ControlPanelsContainer.jsx

```diff
@@ -198,7 +198,10 @@ class ControlPanelsContainer extends React.Component {
     const querySectionsToRender = [];
     const displaySectionsToRender = [];
     allSectionsToRender.forEach(section => {
+      // if at least one control in the secion is not `renderTrigger`
+      // or asks to be displayed at the Data tab
       if (
+        section.tabOverride === 'data' ||
```

Comment:
The lines 208-210 below should be changed too right (could be in another PR so we can fix this soon)? To account for control potentially being a string or an object.

Reply:
Yes, I think we should preprocess the configs at some point and avoid accounting for two types. I'll see what I can do later.